exchanges/stream/websocket_connection.go

```diff
@@ -326,5 +321,18 @@ func (w *WebsocketConnection) SendMessageReturnResponses(ctx context.Context, si
 		w.Reporter.Latency(w.ExchangeName, outbound, time.Since(start))
 	}
 
+	if request.IsVerbose(ctx, w.Verbose) {
```

Comment:
Would you consider the following check here? 
```suggestion
if request.IsVerbose(ctx, w.Verbose) && expected > 1 {
```

If we are already verbose, the request and return message is already going to be logged. I definitely see the benefit to identifying SMRR together, but when its just 1, we already log them